src/components/basic/form/FieldSet.js

```diff
@@ -1,5 +1,28 @@
 import React from "react";
+import PropTypes from "prop-types";
 
-const FieldSet = props => <fieldset>{props.children}</fieldset>;
+const FieldSet = props => {
```

Comment:
codacy wants this line to be

```js
const FieldSet = (props) =>
```

Reply:
Updated locally. Currently working through a terminal error when attempting to commit.

Reply:
Updated and committed.

Reply:
formatting rolled thsi back, which im ok with. I don't want to mess with this. The code review tools are just recommendations and I'll attempt to tweak.